config/settings/development.local.yml

```diff
@@ -0,0 +1 @@
+disable_rbac: false
```

Comment:
This should not be staged as other people might not need it + see my comment on the gitignore.